src/services/collection-service/collection-service.ts

```diff
@@ -68,4 +69,21 @@ export class CollectionService extends CommonResourceService<CollectionResource>
 
     }
 
+    trash(uuid: string): Promise<CollectionResource> {
```

Comment:
Could you create a TrashableResourceService that extends CommonResourceService and implements trash and untrash methods? These methods are copied into groups service. They could be inherited.